arangod/IResearch/IResearchViewCoordinator.cpp

```diff
@@ -356,20 +357,19 @@ arangodb::Result IResearchViewCoordinator::properties(velocypack::Slice const& s
 
     // check link auth as per https://github.com/arangodb/backlog/issues/459
     ExecContext const& exe = ExecContext::current();
-    if (!exe.isSuperuser()) {
```

Comment:
If the user is a superuser then the user should have access to the collection. Therefore removing this short-cut as Superuser is not well-defined.